src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/PanelStatus.tsx

```diff
@@ -1,35 +1,58 @@
+import { useContext } from "react";
 import { useSelector } from "react-redux";
 import { getIsIndexed } from "ui/reducers/app";
-import { AnalysisError, AnalysisPayload } from "ui/state/app";
+import { AnalysisPayload } from "ui/state/app";
 import { getExecutionPoint } from "../../../selectors";
+import { HitsContext } from "../../Editor/Breakpoints/Panel/Panel";
+
+const createStatusString = (
+  hits: number | null,
+  executionPoint: string | null,
+  analysisPoints: AnalysisPayload
+) => {
+  const denominator = hits;
+  let numerator = "…";
```

Comment:
What does a status string like "…/2" mean? Should we maybe just return "0/2" in that case? or even just "…"?

Reply:
I would vote `...` or `?/2`